build.gradle

```diff
@@ -8,7 +8,7 @@ buildscript {
 plugins {
     id 'com.android.application' version '7.1.2' apply false
     id 'com.android.library' version '7.1.2' apply false
-    id 'org.jetbrains.kotlin.android' version '1.6.10' apply false
+    id 'org.jetbrains.kotlin.android' version '1.6.21' apply false
```

Comment:
Why did we need to upgrade the version?

Reply:
@Celinarabe, that is odd. I don't remember updating this. I will keep it at the version it we were initially using. Thanks for the catch!